website/static/js/fangorn.js

```diff
@@ -1011,7 +1015,7 @@ function _fangornDropzoneError(treebeard, file, message, xhr) {
     if (file.isDirectory) {
         msgText = 'Cannot upload folders.';
     } else if (xhr && xhr.status === 507) {
-        msgText = 'Cannot upload file due to insufficient storage.';
+        msgText = 'This file cannot be added to this project because it would exceed the storage limit for OSF Storage';
```

Comment:
"This file cannot be added to this project/component because it would exceed the storage limit for OSF Storage"